test/engine.spec.ts

```diff
@@ -714,3 +714,29 @@ describe('dateTime', () => {
     expect(engine.numberToTime(1.1)).toEqual({'hours': 26, 'minutes': 24, 'seconds': 0})
   })
 })
+
+describe('engine should build correctly', () => {
+  it('should build correctly #1', () => {
```

Comment:
Can we describe the tests a little bit more? Examples are self-explanatory, they should work. Byt why? when? Should we care when we do a refactor next year?

```suggestion
describe('Graph dependency order module blah blah', () => {
  it('should build correctly when rows are dependant on cells that are not yet processed', () => {
```

This will then be contacted to:
"Graph dependency order module should build correctly when rows are dependant on cells that are not yet processed'"

I'm not sure if I'm describing this test correctly. But that's exactly my point, I don't know what it's for.